### DISCUSSION TITLE
GitHub Copilot could not connect to the server - authentication timeout

### DISCUSSION THREAD

**Discussion by mfisco** [community/community#15810] [Copilot Conversations] [answered]
I've been using Copilot for some time now (macOS Monterey 12.3.1 (21E258)) and also suddenly started having a similar problem to https://github.com/github/feedback/discussions/15791#discussion-4042654.  I **do have access** to the beta and I've tried the following:
1. Uninstalling/Reinstalling vscode
2. Uninstalling/Reinstalling the extension
    > When reinstalling I prompted to sign in _via github_ when I install the extension. I can only sign-in with the lower left profile icon with my _connected microsoft account_ but for some reason it's not registering the connected github account. 
3. Turning on/off my vpn
4. Adding the Github extension with a personal access token as suggested above

Please advise on how to fix!

<img width="1362" alt="image" src="https://user-images.githubusercontent.com/67477435/165995798-d61f6a9a-f27b-410a-8482-0d68e76b545e.png">

![image](https://user-images.githubusercontent.com/67477435/165993299-4dbcd2b6-4eff-417c-8cf7-55284c3b6f43.png)

**Comment by davecheney** [comment, marked as answer]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by mfisco** [reply]
Thanks for getting back to me Dave. Turns out it had to do with the builtin github auth extension. I was able to get it working doing the following from https://github.com/microsoft/vscode/issues/144448#issue-1160183783:
1. Uninstalled _Github Copilot_
2. In the extensions view, entered `@builtin github` and enabled the _Github Authentication_ (somehow it became disabled but i'm not sure how since I never did so explicitly)
4. Re-installed  _Github Copilot_


Anyways, appreciate your help and thanks for your time.

**Reply by LeboulangerSimon** [reply]
Nice ! It works for me too ! Thanks a lot !
